UE/Application/Ports/BtsPort.cpp

```diff
@@ -80,4 +85,12 @@ void BtsPort::sendAttachRequest(common::BtsId btsId)
 
 }
 
+void BtsPort::handleMessageSend(Sms &sms)
+{
+    //logger.logInfo("sms send from:"+sms.from+" to:"+sms.to);
```

Comment:
Why is that commented?
```suggestion
    logger.logInfo("sms send from:" + sms.from + " to:" + sms.to);
```